Header header
bool lineOneFound
bool lineTwoFound
float64[] lineOne
float64[] lineTwo
float64 angle
